lib/util.js

```diff
@@ -517,7 +517,7 @@ function formatCollectionIterator(ctx, value, recurseTimes, visibleKeys, keys) {
   var nextRecurseTimes = recurseTimes === null ? null : recurseTimes - 1;
   var vals = mirror.preview();
   var output = [];
-  for (let o of vals) {
+  for (const o of vals) {
```

Comment:
Things like this do make me wonder if this is a wise rule to implement after all. The main argument for using `const` is that it lessens the cognitive load. You *know* that variable won't change. Except...uh...yeah....this. I think this actually makes the code *harder* to understand.